ts/Extensions/Breadcrumbs.ts

```diff
@@ -375,15 +307,14 @@ class Breadcrumbs {
      * Properties
      *
      * */
-    public chart: Chart = void 0 as any;
     public breadcrumbsGroup: SVGElement = void 0 as any;
     public breadcrumbsList: Array<Array<any>> = [];
     public isDirty: boolean = true;
     public isTreemap?: boolean = void 0 as any;
     public level: number = -1;
-    public options: Highcharts.BreadcrumbsOptions = void 0 as any;
+    public options: BreadcrumbsOptions = void 0 as any;
 
-    public constructor(chart: Chart, userOptions?: Highcharts.BreadcrumbsOptions, isTreemap?: boolean) {
+    public constructor(chart: Chart, userOptions?: BreadcrumbsOptions, isTreemap?: boolean) {
```

Comment:
BreadcrumbsOptions have defaults, not used by users. Therefor user options should be of type `DeepPartial<BreadcrumbsOptions>`.